### ISSUE TITLE
html5spec update frequency

### ISSUE THREAD

**Issue by tenzap** [validator/validator#2123]
### Summary

Not that I need the html5spec to be updated, but it seems it hasn't been updated for 4 months. Is this normal? I remember it was planed to have a job that creates a PR that refreshes the spec at almost every commit.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
N/a
```

### Checker error messages

```shell
N/a
```

### Steps to reproduce

1.
2.
3.
n/a 

### Expected behavior

n/a 

### Actual behavior

!/a

**Comment by sideshowbarker** [comment]
oop — thanks for catching that. As far as I can see now, it _never_ actually worked even from the beginning when we first set it up. But it should start working as expected now. I think.
